ssntp/README.md

```diff
@@ -97,6 +97,31 @@ UUIDs.
 3. Connection is successfully established. Both ends of the connection
    can now asynchronously send SSNTP frames.
 
+## CIAO Cert ##
+
+SSNTP uses ciao-cert to generate the certificates it needs to communicate. They
+can be generated as follows.
+
+```
+sudo mkdir -p /etc/pki/ciao/
+sudo mkdir -p /var/lib/ciao/logs/scheduler
+sudo -E $GOPATH/bin/ciao-cert -directory /etc/pki/ciao -host localhost -server -role scheduler
```

Comment:
I agree.

It will be easier to maintain if ssntp docu notes that it depends on certificates with roles embedded and links to the docu on creating those, eg:

https://github.com/01org/ciao/blob/master/ciao-cert/README.md
https://godoc.org/github.com/01org/ciao/ciao-cert

(note the second of those is empty...maybe that's the place to put some attention)